sdk/storage/azure-storage-blob/azure/storage/blob/_download.py

```diff
@@ -365,6 +365,11 @@ def __init__(
 
         self._response = self._initial_request()
         self.properties = self._response.properties
+
+        # TODO: This is definitely not how we want to do this
+        if('x-ms-encryption-context' in self._response.response.headers):
+            self.properties['encryption_context'] = self._response.response.headers['x-ms-encryption-context']
+
```

Comment:
Just want to call this out that this is odd that we would have something DataLake-specific in the Blob code (because DataLake calls into Blob)

Reply:
EV outfront

Reply:
TODO: Try doing this operation in Dlake's `_download.py` instead